package.json

```diff
@@ -19,17 +19,20 @@
         "styles"
     ],
     "scripts": {
+        "prepare": "husky install",
```

Comment:
husky doesn't install hooks automatically, husky documentation recommends use `prepare` script, but I'm not sure. `prepare` is executed during install, pack and publish.

Reply:
I don't think it is a problem